Okta.Xamarin/Okta.Xamarin/OktaStateManager.cs

```diff
@@ -28,12 +26,12 @@ public OktaStateManager()
         /// <summary>
         /// Initializes a new instance of the <see cref="OktaStateManager"/> class.
         /// </summary>
-        /// <param name="accessToken"></param>
-        /// <param name="tokenType"></param>
-        /// <param name="idToken"></param>
-        /// <param name="refreshToken"></param>
-        /// <param name="expiresIn"></param>
-        /// <param name="scope"></param>
+        /// <param name="accessToken">The access token.</param>
+        /// <param name="tokenType">The token type.</param>
+        /// <param name="idToken">The ID token.</param>
+        /// <param name="refreshToken">The refresh token.</param>
+        /// <param name="expiresIn">Expires in.</param>
+        /// <param name="scope">Scope.</param>
         public OktaStateManager(string accessToken, string tokenType, string idToken = null, string refreshToken = null, int? expiresIn = null, string scope = null)
```

Comment:
Same here, tokenType vs tokenKind. SHould we use just one name for consistency?

Reply:
This one actually is the tokenType value returned by the server.  For now it is recorded in the StateManager to advise what would be the appropriate Authorization header value prefix (see previous comment on this topic); this is my reasoning thus far.  Usage will reveal whether this is useful.